README.md

```diff
@@ -40,7 +40,7 @@ ant run
 This also creates locally a .jar file, that you can distribute and use on other machines.
 
 ## Testing Logisim-evolution
-As Logisim-evolution needs udpates (new features and patches) and that unit tests are not implemented within the code, the *testing* branch was created.
+As Logisim-evolution needs updates (new features and patches) and that unit tests are not implemented within the code, the *testing* branch was created.
```

Comment:
I would also suggest to fix the second part of the sentence:

"As Logisim-evolution needs updates (new features and patches) and _currently lacks unit tests_, the *testing* branch was created."

Reply:
Nice idea, done!